addons/reorder-custom-inputs/addon.json

```diff
@@ -1,12 +1,18 @@
 {
   "name": "Rearrangeable custom block inputs",
-  "description": "Adds buttons for rearranging custom block inputs.",
+  "description": "Allows rearranging custom block parameters with the editor's \"Make a block\" screen.",
   "credits": [
     {
       "name": "Chrome_Cat",
       "link": "https://scratch.mit.edu/users/Chrome_Cat/"
     }
   ],
+  "info": [
+    {
+      "id": "editCustomBlock",
+      "text": "To edit a custom block, right click it and choose the edit option."
+    }
+  ],
```

Comment:
I feel like saying this here might make it sound like this is a feature we've added or something.

Reply:
I agree. There is no need to include info on how to use a basic feature of Scratch.